db/seeds.rb

```diff
@@ -91,6 +91,23 @@
 CompletedLearningUnit.create(user: ale, learning_unit: ruby)
 CompletedLearningUnit.create(user: ale, learning_unit: js)
 
+paid = Label.create(name: "Paid", description: "This resource is paid")
+media = Label.create(name: "Media", description: "Includes a video")
+article = Label.create(name: "Article", description: "Is an article")
+
+
+[github, ruby_udemy, react_redux].each do |resource|
+  ResourceLabel.create(resource_id: resource.id, label_id: paid.id)
```

Comment:
Remember that you can write this also as `ResourceLabel.create(resource: resource, label: paid)`, which is a little bit concise.

Reply:
Yes! However, I tried to follow DRY and avoid repeat each line on any new resource label creation.

Reply:
Done!